### PR TITLE
feat(working on notification)

### PR DESCRIPTION
**Pull request by Ruthuwamahoro**
#### What does this PR do?

-Resolving conflicts between chat app and notification

#### Description of Task to be completed?

-Ensure that chat app and chat share the same server
-ensure that chat app is working correctly
-Ensure that the notification app is working

#### How should this be manually tested?

- set up the env according to your localhost database credentials
- use the api in the index.ts in routers to connect to the database
- use npm run dev to connect to the database

#### Any background context you want to provide?

-dummy data are provided before having the real data

### CODE REVIEW

**Review by musabehonore — APPROVED** [review]

**Review by Sophie20202 — APPROVED** [review]

**Review by RWEMAREMY — APPROVED** [review]
Good

File: public/index.html [hunk 1]
```diff
@@ -233,7 +233,7 @@ <h3 id="welmsg"></h3>
       const establishSocketConnection = () => {
         const tokenValue = localStorage.getItem('Token');
         if (tokenValue) {
-          const socket = io('https://alpha-ec-be.onrender.com/', {
+          const socket = io('http://localhost:3000/', {
```
Comment (fab-ryan) [inline public/index.html]: change this please

File: public/index.html [hunk 2]
```diff
@@ -253,17 +253,14 @@ <h3 id="welmsg"></h3>
...
-              {
-                method: 'GET',
-                headers: {
-                  'Content-Type': 'application/json',
-                  Accept: 'application/json',
-                  Authorization: `Bearer ${localStorage.getItem('Token')}`,
-                },
-              }
-            );
+            const messages = await fetch('http://localhost:3000/api/chats', {
```
Comment (fab-ryan) [inline public/index.html]: same as here

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Please remove that hard coded url

**Review by fab-ryan — APPROVED** [review]

**Comment by Ruthuwamahoro** [comment]
> Please remove that hard coded url

done

**Pull request merged**